tests/containers/netavark_integration.pm

```diff
@@ -55,7 +55,7 @@ sub run {
 
     # Install tests dependencies
     my @pkgs = qw(aardvark-dns cargo firewalld iproute2 iptables jq make protobuf-devel netavark);
-    if (is_tumbleweed || is_microos) {
+    if (is_tumbleweed || is_sle('>=16.0')) {
```

Comment:
I think `is_microos` got accidentally removed

Reply:
We do not run these on micro, or do we?

Reply:
Not really :)